Sofa/Component/ODESolver/Backward/src/sofa/component/odesolver/backward/EulerImplicitSolver.cpp

```diff
@@ -138,15 +138,15 @@ void EulerImplicitSolver::solve(const core::ExecParams* params, SReal dt, sofa::
         // new more powerful visitors
 
         // force in the current configuration
-        b.eq(f,1.0/tr);                                                                         // b = f0
+        b.eq(f,1.0/tr);                                                                    // b = f
 
         msg_info() << "EulerImplicitSolver, f = " << f;
 
         // add the change of force due to stiffness + Rayleigh damping
-        mop.addMBKv(b, -d_rayleighMass.getValue(), 1, h + d_rayleighStiffness.getValue()); // b =  f0 + ( rm M + B + (h+rs) K ) v
+        mop.addMBKv(b, -d_rayleighMass.getValue(), 0, h + d_rayleighStiffness.getValue()); // b =  f + ( rm M + (h+rs) K ) v
```

Comment:
B the way I don't understand why the minus sign on the mass matrix. This is also strange. My understanding is that the sign should be the same for both Rayleigh factors.

Reply:
@bakpaul @hugtalbot seems to be SOFA's convention for mass and force field. I've documented it with the good signs in Caribou: https://github.com/mimesis-inria/caribou/blob/master/src/SofaCaribou/Ode/BackwardEulerODESolver.cpp#L107

Reply:
@hugtalbot no idea, but it doesn't change anything. However, if you want to invert it, you will have to update every forcefields of SOFA so that they remove the minus factor. This is not always as straightforward [as in caribou](https://github.com/mimesis-inria/caribou/blob/master/src/SofaCaribou/Forcefield/HyperelasticForcefield.inl#L166), sometime the minux factor is hidden into the displacement computation (see tetrahedronFemForcefield for an example if I remember correctly).